### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by steffengr**
This PR fixes the structure and navigation to prepare the addition of improved documentation.

### CODE REVIEW

File: docs/integrations/guides/databricks.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    :scale: 30 %
+    :figclass: align-center
+
+**Installing hopsworks-cloud-sdk**
+
+The feature store library needs to be installed to connect to it. In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries* and then *Install New*. As *Library Source* choose *PyPI* and fill in *hopsworks-cloud-sdk* into the *Package* field.
+
+**Creating a DBFS folder to store certificates**
+
+Hopsworks relies on certificates being available in the Databricks cluster to connect to some services inside Hopsworks. To ensure that these certificates can be distributed to all nodes in a Databricks cluster, Hopsworks relies on them being stored in the `Databricks file system <https://docs.databricks.com/data/databricks-file-system.html>`_.
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: Hopsworks relies -> The Hopsworks Feature Store relies

File: docs/integrations/guides/databricks.rst [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+For this purpose, you should create a cert folder in DBFS. This can be done by running the following in a Databricks notebook connected to the prepared cluster::
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: cert folder -> certs folder
running the following -> executing the following statement

File: docs/integrations/guides/databricks.rst [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+.. _feature-store-connect-databricks.png: ../../_images/feature-store-connect-databricks.png
+.. figure:: ../../imgs/cloud/feature-store-connect-databricks.png
+    :alt: Connect to feature store from Databricks
+    :target: `feature-store-connect-databricks.png`_
+    :align: center
+    :scale: 35 %
+    :figclass: align-center
+
+In the Databricks notebooks connected to the prepared cluster, use the following commands to connect to the feature store (see screenshot above for how to change 'myinstance' to the hostname of your Hopsworks feature store and how to change 'my_project' to the project containing your feature store)::
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: commands -> statements

File: docs/integrations/guides/databricks.rst [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+    import hops.featurestore as fs
+    fs.connect('my_instance', 'my_project', region_name='my_aws_region', cert_folder='/dbfs/certs/')
+
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: replace by -> replace with

File: docs/integrations/guides/databricks.rst [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+| Replace *my_instance* by the address of your instance, *my_project* by the name of your project and *my_aws_region* by the AWS region in which you stored the API Key. |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+If you have trouble connecting, then ensure that the Security Group of your Hopsworks instance on AWS is configured to allow incoming traffic from your Databricks clusters. See `VPC Security Groups <https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html>`_. for more information. If the Hopsworks instance is not accessible from the internet then you will need to configure `VPC Peering <https://docs.databricks.com/administration-guide/cloud-configurations/aws/vpc-peering.html>`_.
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: internet -> Internet,

File: docs/integrations/guides/databricks.rst [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+**Installing hops library**
+
+.. warning:: 
+ - This is an advanced setup, if you are not sure that you need it or if you are not sure of what you are doing, please use hopsworks-cloud-sdk. 
+ - Hopsworks-cloud-sdk needs to be able to connect directly to the ip on which the hops name node and hive metastore are listening. This means that if you deploy Hopsworks on AWS you will either need to start Hopsworks in the same VPC as your Databricks cluster or to set up `VPC Peering <https://docs.databricks.com/administration-guide/cloud-configurations/aws/vpc-peering.html>`_ between your Databricks cluster and the Hopsworks one.
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: hops name node -> Hops Name Node
hive metastore -> Hive Metastore

File: docs/integrations/guides/databricks.rst [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+
+In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries*. Make sure that hopsworks-cloud-sdk is not installed, make sure to uninstall it if that's the case. Then *Install New*. As *Library Source* choose *PyPI* and fill in *hops* into the *Package* field.
+
+**Setting up the cluster to use hops**
+
+After installing the hops library, restart the cluster and open a Databrick notebooks connected to this cluster. Run the following command in this notebook::
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: statements

Comment (jimdowling) [reply docs/integrations/guides/databricks.rst]: run -> Execute

File: docs/integrations/guides/databricks.rst [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Replace *my_instance* by the address of your instance, *my_project* by the name of your project and *my_aws_region* by the AWS region in which you stored the API Key. |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+This will return two configurations that you need to add to your cluster configuration. To set them go to *Clusters*, select your cluster and click *Edit*. 
+Then, select *Configuration* and *Advanced Options*. 
+
+For the first configuration select *Init Scripts* and copy the init script path returned by *setup_databricks* in *Init Script Path* and click *Add*.
+
+For the second configuration select *Spark* and copy the spark configuration returned by *setup_databricks* in *Spark Config*.
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: For the .. configuration -> For the .. configuration,

File: docs/integrations/guides/databricks.rst [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+
+Once you have set these two configurations, click *Confirm and Restart*.
+
+**Connecting to the Feature Store**
+
+In the Databricks notebooks connected to the prepared cluster use the following code to connect to the feature store::
```
Comment (jimdowling) [inline docs/integrations/guides/databricks.rst]: prepared cluster ->  prepared cluster,

File: docs/integrations/guides/sagemaker.rst [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+.. figure:: ../../imgs/cloud/aws_policy.png
+    :alt: Hopsworks feature store set policy
+    :target: `hopsworks_aws_policy.png`_
+    :align: center
+    :scale: 30 %
+    :figclass: align-center
+
+**Installing hopsworks-cloud-sdk and connecting to the Feature Store**
+
+To be able to access the Hopsworks Feature Store, the hopsworks-cloud-sdk library needs to be installed. One way of achieving this is by opening a Python notebook in SageMaker and installing the latest hopsworks-cloud-sdk. Note that the library will not be persistent. For information around how to permanently install a library to Sagemaker see `Install External Libraries and Kernels in Notebook Instances <https://docs.aws.amazon.com/sagemaker/latest/dg/nbi-add-external.html>`_. ::
```
Comment (jimdowling) [inline docs/integrations/guides/sagemaker.rst]: install a library to Sagemaker ->  install a library to Sagemaker,

File: docs/integrations/guides/databricks.rst [hunk 11]
```diff
@@ -0,0 +1,160 @@
+Integrating with Databricks
```
[the remark this answers is not in the document]
Comment (steffengr) [reply docs/integrations/guides/databricks.rst]: Don't focus on this I will redo the entire sections. I just put it in the right place for now.

**Comment by steffengr** [comment]
@jimdowling I fixed all your comments

**Comment by tkakantousis** [comment]
@steffengr can you send screenshots or a gif to see the updates?

File: docs/featurestore/featurestore.rst [hunk 12]
```diff
@@ -1018,11 +846,11 @@ Instructions for installing Hopsworks (which comes with the feature store) on AW
 
 **How can I use Hopsworks Featurestore from Amazon Sagemaker?**
 
-Documentation for accessing Hopsworks Featurestore from Amazon Sagemaker is available here: `https://hopsworks.readthedocs.io/en/latest/user_guide/hopsworks/featurestore.html#connecting-from-amazon-sagemaker <https://hopsworks.readthedocs.io/en/latest/user_guide/hopsworks/featurestore.html#connecting-from-amazon-sagemaker>`_.
+Documentation for accessing Hopsworks Featurestore from Amazon Sagemaker is available here: Databricks_.
 
 **How can I use Hopsworks Featurestore from Databricks Platform?**
 
-Documentation for accessing Hopsworks Featurestore from Databricks Platform is available here: `https://hopsworks.readthedocs.io/en/latest/user_guide/hopsworks/featurestore.html#connecting-from-databricks-notebooks <https://hopsworks.readthedocs.io/en/latest/user_guide/hopsworks/featurestore.html#connecting-from-databricks-notebooks>`_.
+Documentation for accessing Hopsworks Featurestore from Databricks Platform is available here: SageMaker_.
```
Comment (tkakantousis) [inline docs/featurestore/featurestore.rst]: @steffengr sagemaker link in databricks section and vice versa

**Comment by steffengr** [comment]
@tkakantousis it's better if you build it locally. The changes are partially interactive. Start on the main page. I changed the categories and sections. Then go to the feature store to check out the new nav bar.

File: docs/featurestore/featurestore.rst (hunk 12 above)
Comment (steffengr) [reply docs/featurestore/featurestore.rst]: Thanks I fixed that

**Pull request merged**